specifyweb/frontend/js_src/lib/components/InitialContext/remotePrefs.ts

```diff
@@ -189,6 +189,13 @@ export const remotePrefsDefinitions = f.store(
         parser: 'java.lang.Long',
         isLegacy: true,
       },
+      'TaxonTreeEditor.DisplayAuthor' : {
```

Comment:
Can we change this to accept a rank id instead?
I.e, display authors for all nodes in the tree at that rank id or further down.

Reasoning:
People might want to customize this for performance reasons.
People might want to customize this for purpose of reducing the visual clutter in the tree viewer

Reply:
Yes I can change it to accept a rankid. Could we change this to accept a rank name instead? That may be more intuitive to users than an id. 
Currently, the first thought/easiest way I had of getting the rankid would probably be to setup a query like this. Even this may be too complicated to figure out.
![taxonRankQuery](https://user-images.githubusercontent.com/64045831/212487340-5d45ee19-4a14-4b4a-80ca-141fe8c41cc7.png)

How much do we assume for users that can modify the remote preferences?

Reply:
What would we want the default for this to be? 180 (Genus)? Or 220 (Species)?
Is it possible these rankids change if there are custom ranks?

Reply:
I suggested using rank id only because it's already used by other remote prefs. Maybe because with rank IDs you can easily do a `>` comparison to see if a node is above or below the fold. Since rank id is a number, it also doesn't have case sensitivity problems. As far as getting rank id, I think Specify is using standard IDs for them that would be familiar to techy biologists.

It's your choice whether you want to use rank id or rank name

Have to ask @grantfitzsimmons for what the default value should be.

Reply:
Since we are changing the functionality, perhaps the name of the pref could be changed as well. 
How about something like `TaxonTreeEditor.DisplayAuthor.ForRanks`? 
I am open for ideas. If the current name is ok, then no need to chang it.

Reply:
Author should be shown for Genus and below by default

Reply:
Current name is fine.
`TaxonTreeEditor.DisplayAuthor.ForRanks` is misleading as you only specify one rank id as a value.
`TaxonTreeEditor.DisplayAuthor.UnderRanks ` may be better expressed as `TaxonTreeEditor.DisplayAuthor.BelowRanks` - feel free to rename it if you want